iterx.go

```diff
@@ -19,17 +19,14 @@ var DefaultUnsafe bool
 
 // Iterx is a wrapper around gocql.Iter which adds struct scanning capabilities.
 type Iterx struct {
+	err error
 	*gocql.Iter
-	Mapper *reflectx.Mapper
-
+	Mapper     *reflectx.Mapper
+	fields     [][]int
+	values     []interface{}
 	unsafe     bool
 	structOnly bool
 	applied    bool
-	err        error
-
-	// Cache memory for a rows during iteration in structScan.
-	fields [][]int
-	values []interface{}
```

Comment:
Are we okay with losing the information that this comment gives?

Reply:
thanks for pointing out, fixed.